docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -2,263 +2,243 @@
+print(f" > Expectation value: {result.values[0]}")
+print(f" > Metadata: {result.metadata}")
 ```
 
-#### Modify [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) options
-
-If you want to keep some configuration values for several runs, it can be better to change the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) options. That way you can use the same [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) object as many times as you wish without having to
-rewrite the configuration values every time you use [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run).
-
-#### Modify existing [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator)
-
-If you prefer to change the options of an already-defined [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator), you can use the method [`qiskit.primitives.Estimator.set_options`](../api/qiskit/qiskit.primitives.Estimator#set_options) and introduce the new options as keyword arguments.
-
-```python
-estimator.set_options(shots=2048, seed=123)
-
-job = estimator.run(qc, observable)
-result = job.result()
-print(result)
+```text
+ > Expectation value: 3.999999999999999
+ > Metadata: [{}, {}, {}]
 ```
+  </TabItem>
+</Tabs>
 
-```python
-EstimatorResult(values=array([4.]), metadata=[{'variance': 3.552713678800501e-15, 'shots': 2048}])
-```
+### Set Estimator run options
 
-```python
-print(result.values[0])
-```
+By default, the reference Estimator performs an exact statevector calculation based on the
+[`quantum_info.Statevector`](../api/qiskit/qiskit.quantum_info.Statevector) class.
+However, this can be modified to introduce the effect of the sampling overhead (also known as "shot noise").
 
-```python
-3.999999998697238
-```
+With the introduction of the new V2 interface, the sampling overhead in the Estimator is now more concretely defined.
+The new interface accepts a `precision` argument that expresses the error bars that the
+primitive implementation should target for expectation values estimates,
+instead of the number of `shots` used in the V1 interface.
 
-#### Define a new [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) with the options
+<Tabs>
+  <TabItem value="V2" label="StatevectorEstimator (V2)">
 
-If you prefer to define a new [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) with new options, define a `dict` like this one:
+      The sampling overhead is now called `precision`, and is defined exclusively at the `.run()`
+      method level. This allows for a more finely-grained tuning of the option all the way to the PUB level.
 
-```python
-options = {"shots": 2048, "seed": 123}
-```
+    ```python
+    # Circuits and parameters must obey the Instruction Set Architecture (ISA) of a particular backend.
+    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
```

Comment:
Given that circuit 1 and 2 don't "exist" (haven't been defined), and we already have the ISA explanation before. In this case I would jump directly to showing the `estimator.run()`:

```suggestion
```